dp_mobility_report/visualization/plot.py

```diff
@@ -283,9 +285,12 @@ def choropleth_map(
         cmap = mpl.cm.PiYG #RdBu_r
         norm = mpl.colors.TwoSlopeNorm(vmin=vmin, vcenter=0, vmax=vmax)
     else:
-        cmap = mpl.cm.YlOrRd
+        if not max_scale:
+            max_scale = counts_per_tile_gdf[fill_color_name].max()
+        if not cmap:
+            cmap = mpl.cm.YlOrRd
```

Comment:
I changed this function a little to be more generic

https://github.com/FreeMoveProject/dp_mobility_report/commit/6f47361e5a8a748ce0d5d526c354f77f2979ba9e